js/pulse.js

```diff
@@ -252,46 +257,89 @@
                 return '0,0,0';
             }
 
-            function getByPriority(){
-                for (var i = 0,n = arguments.length; i < n; i++){
-                    if(arguments[i] || arguments[i] === ""){
+            function getByPriority() {
+                for (var i = 0,n = arguments.length; i < n; i++) {
+                    if(arguments[i] || arguments[i] === "") {
                         return arguments[i];
                     }
                 }
                 //If none of the passed objects exist
                 return null;
             }
 
-            function getProtocolFromPulseHost(host){
-                if(host.indexOf("https") === 0){ //Then it starts with https
+            function getProtocolFromPulseHost(host) {
+                if(host.indexOf("https") === 0) { //Then it starts with https
                     return "https://"
                 } else {
                     return "http://"
                 }
             }
 
-            function getPulseAccount(host){
+            function getPulseAccount(host) {
                 var regEx = /(?:https?:\/\/)?(.*)\/?/;
                 return host.match(regEx)[1];
 
             }
 
-            function getCategoryFromPlayerLevelShares(shares){
+            function getCategoryFromPlayerLevelShares(shares) {
                 //Category is the first element
                 var values = safeSplit(shares,",");
-                if(values && values.length !== 0){
+                if(values && values.length !== 0) {
                     return values[0];
                 }
             }
 
-            function getContentPartnerFromPlayerLevelShares(shares){
+            function getContentPartnerFromPlayerLevelShares(shares) {
                 //Category is the first element
                 var values = safeSplit(shares,",");
-                if(values && values.length === 2){
+                if(values && values.length === 2) {
                     return values[1];
                 }
             }
 
+            var updateAdScreenPointerEventsEnabled = (function() {
+                var adScreens = document.getElementsByClassName('oo-ad-screen');
+                var skinClickLayers = document.getElementsByClassName('oo-player-skin-plugins-click-layer');
+
+                if(adScreens.length === 0) {
```

Comment:
`getElementsByClassName` will return an empty `HTMLCollection` if no matching elements are found, so it should be fine.